app-nodejs-html5/package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "app-nodejs-html5",
+  "version": "0.0.1",
+  "description": "HTML5 Service Invocation Application",
+  "main": "app.js",
```

To use ES module syntax you can mark set the type of this package to `module`.
```suggestion
  "main": "app.js",
  "type": "module",
```

---

@jonkoops can you confirm that RH SSO prefers to use the ES module syntax over the CommonJS?  There are still some problems in using ES (for example APM) such that we've not moved most of the Node.js team assets over.